ci/Dockerfile-envoy-windows

```diff
@@ -3,9 +3,12 @@ ARG BUILD_TAG=ltsc2019
 
 FROM $BUILD_OS:$BUILD_TAG
 
+ARG BUILD_TYPE=release
+
 RUN mkdir "C:\\Program\ Files\\envoy"
 RUN setx path "%path%;c:\Program Files\envoy"
-ADD ["windows/amd64/envoy.exe", "C:/Program Files/envoy/"]
+ADD ["windows/amd64/envoy.*", "C:/Program Files/envoy/"]
+RUN if "%BUILD_TYPE%" == "release" del "C:\Program Files\envoy\envoy.pdb"
```

Comment:
Won't this still leave envoy.pdb in docker layers if the image isn't squashed?

Reply:
Yeah, I think you are right. I don’t know of a way to conditionally do an ADD, might have to change it to how we’ve done it on Linux with two different directories to ADD from between debug and regular image builds

Reply:
In Linux build we setup two directory and use directory name to change what file to add.